src/components/Dialog.jsx

```diff
@@ -18,59 +18,69 @@ class Dialog extends React.Component {
     // }
   }
 
+  stopEvent(e) {
+    e.preventDefault && e.preventDefault();
+    e.stopPropagation && e.stopPropagation();
+    e.returnValue = false;
+    e.cancelBubble = true;
+    return false;
+  }
+
+  close(e) {
+    this.onClose();
+    return this.stopEvent(e);
+  }
+
   render() {
     const width = 800;
     const height = 425;
-    const x = window.innerWidth / 2 - (width / 2);
-    const y = window.innerHeight / 2 - (height / 2) + window.pageYOffset;
+    const x = (window.innerWidth / 2) - (width / 2);
+    const y = (window.innerHeight / 2) - ((height / 2) + window.pageYOffset);
```

Comment:
Why is `window.pageYOffset` set within the brackets? This will cause the maths to expand so that it becomes `window.innerHeight / 2 - height / 2 MINUS window.pageYOffset`, so instead of having the dialog more-or-less anchored to the centre of the browser window, _it causes the dialog to appear higher in the page the lower down you've scrolled._